maps/randomvaults/objects.dm

```diff
@@ -174,7 +177,7 @@
 
 /obj/machinery/power/apc/no_alerts/vault_listening/initialize()
 	. = ..()
-	name = "\improper Listening Outpost APC."
+	name = "\improper Listening Outpost APC"
```

Comment:
And yet if we decide to roll back this PR because we want to undo the vault changes, this change will also be rolled back, and the changes aren't related enough for that. Err on the side of caution, instead of trying to push the definition of "related".